tensorrt_llm/serve/openai_server.py

```diff
@@ -354,6 +375,7 @@ async def create_completion_generator(
                     pp_result = request_output.outputs[0]._postprocess_result
                 for pp_res in pp_result:
                     yield pp_res
+            logger.info(f"generation completed for request {raw_request.state.request_id}")
```

Comment:
Oh, logger.info might be too verbose - are you planning to move it to verbose log, or guarded by ENV?

Reply:
This will be only once per request at the completion so not too verbose. The per-iter log is in `py_executor.py`. But we can change this as well if needed to either be guarded or demoted to `debug`